test/models/subscription_status_test.rb

```diff
@@ -17,30 +17,40 @@ def test_get_status
     end
 
     def test_to_status_valid
+      host.organization.stubs(:simple_content_access?).returns(false)
       stub_status(Katello::Candlepin::Consumer::ENTITLEMENTS_VALID)
       assert_equal Katello::SubscriptionStatus::VALID, status.to_status
     end
 
     def test_to_status_disabled
+      host.organization.stubs(:simple_content_access?).returns(false)
       stub_status(Katello::Candlepin::Consumer::ENTITLEMENTS_DISABLED)
       assert_equal Katello::SubscriptionStatus::DISABLED, status.to_status
     end
 
     def test_to_status_partial
+      host.organization.stubs(:simple_content_access?).returns(false)
       stub_status(Katello::Candlepin::Consumer::ENTITLEMENTS_PARTIAL)
       assert_equal Katello::SubscriptionStatus::PARTIAL, status.to_status
     end
 
     def test_to_status_invalid
+      host.organization.stubs(:simple_content_access?).returns(false)
       stub_status(Katello::Candlepin::Consumer::ENTITLEMENTS_INVALID)
       assert_equal Katello::SubscriptionStatus::INVALID, status.to_status
     end
 
     def test_to_status_unsubscribed_hypervisor
-      stub_status('unsubscribed_hypervisor')
+      host.subscription_facet.stubs(:unsubscribed_hypervisor?).returns(true)
       assert_equal Katello::SubscriptionStatus::UNSUBSCRIBED_HYPERVISOR, status.to_status
     end
 
+    def test_sca_enabled_hypervisor
+      host.subscription_facet.stubs(:unsubscribed_hypervisor?).returns(false)
+      stub_status(Katello::Candlepin::Consumer::ENTITLEMENTS_DISABLED)
```

Comment:
is stub_status needed? ie does the code path which is getting stub get reached?

Reply:
The `stub_status` is needed because it fails with a vcr error, looking at pry it fails on this line unless I stub it:

https://github.com/Katello/katello/pull/9071/files#diff-746cf731242664a742efafbdcd690f3bceb15b60d66a320d25e3f8aeea73ff3cR48